src/data/complex/exponential.lean

```diff
@@ -1351,6 +1352,18 @@ begin
   convert exp_bound hxc hn; norm_cast
 end
 
+lemma exp_bound' {x : ℝ} (h1 : 0 ≤ x) (h2 : x ≤ 1) {n : ℕ} (hn : 0 < n) :
+  real.exp x ≤ ∑ (m : ℕ) in finset.range n, x ^ m / (m.factorial)
+  + x ^ n * ((n : ℝ) + 1) / (n.factorial * (n : ℝ) ) :=
+begin
+  have h3 : |x| = x, simp, exact h1,
+  have h4 : |x| ≤ 1, rw abs_le, apply and.intro, linarith, linarith,
+  let h' := real.exp_bound h4 hn, rw h3 at h',
```

Comment:
I don't personally use these, but this style of proof suits itself vey much to a `calc` block (you can look up the docs for how to do that!)